decaylanguage/dec/dec.py

```diff
@@ -848,9 +848,9 @@ def find_charge_conjugate_match(pname, dict_cc_names=dict()):
 
     # Search EvtGen names via the Particle class
     try:
-        return Particle.from_dec(pname).invert().name
+        return Particle.from_string(pname).invert().name
     # If anything else fails ...
-    except ParticleNotFound:
```

Comment:
It's generally a bad idea to use `except:`. Can't you list the possible exceptions?

Reply:
This also [catches Control C and system exit!](http://effbot.org/zone/stupid-exceptions-keyboardinterrupt.htm) If you *really* have to catch all, you should at least exclude `KeyboardInterrupt, SystemExit`, but it's *much* better to list the possible exceptions instead.

Reply:
Agreed. Better extend the exception instance I had listed.